server/service.go

```diff
@@ -162,13 +203,22 @@ func (m *BoostService) StartHTTPServer() error {
 
 func (m *BoostService) startBidCacheCleanupTask() {
 	for {
-		time.Sleep(1 * time.Minute)
+		time.Sleep(time.Minute)
 		m.bidsLock.Lock()
 		for k, bidResp := range m.bids {
 			if time.Since(bidResp.t) > 3*time.Minute {
 				delete(m.bids, k)
+				continue
+			}
+			labels := map[string]string{
+				"slot":   fmt.Sprintf("%d", k.slot),
+				"pubkey": bidResp.response.Data.Message.Pubkey.PubkeyHex().String(),
 			}
+			m.metricsMev.gasUsed.With(labels).Set(float64(bidResp.response.Data.Message.Header.GasUsed))
+			m.metricsMev.gasLimit.With(labels).Set(float64(bidResp.response.Data.Message.Header.GasLimit))
+			m.metricsMev.timeSinceBid.With(labels).Set(float64(time.Since(bidResp.t).Milliseconds()))
 		}
+		m.metricsMev.bids.Set(float64(len(m.bids)))
```

Comment:
@metachris i'm not particularly intimate with the bidding process, but had a rough stab at how i think we could export metrics for this. This seemed like the most central place we'd be able to emit metrics from (albeit 1m delayed)

Would be great to hear your thoughts on this

Reply:
also not too sure why the `TestGetPayload/Bad_response_from_relays` test is failing, will look into it after coffee

Reply:
tests fixed